assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -142,17 +149,20 @@ const MediaElement = ({
 
   if (type === 'image') {
     return (
-      <Image
-        key={src}
-        src={src}
-        ref={mediaElement}
-        width={width}
-        height={height}
-        loading={'lazy'}
-        onClick={onClick}
-        dragging={dragging}
-        {...dropTargetsBindings}
-      />
+      <>
+        <Image
+          key={src + id}
```

Comment:
why not just use id here? An id is meant to un unique.

Reply:
100%